dallinger/frontend/static/scripts/dallinger2.js

```diff
@@ -422,12 +422,15 @@ var dallinger = (function () {
     var url = "/participant";
     var data = {};
     var deferred = $.Deferred();
-    if (dlgr.identity.assignmentId) {
+    if (dlgr.identity.entryInformation) {
```

Comment:
This function is such a bug magnet, but I think I follow all the changes and they make sense in the context of the original problem and @pmcharrison's concern about the lost fingerprint.

Reply:
Agreed. I think this is not the time for a full refactoring, but it would be wise at some point. Sending the `entryInformation` any time we have it is probably the right approach.